source/shared/cpp/ObjectModel/SubmitAction.h

```diff
@@ -25,6 +25,10 @@ namespace AdaptiveSharedNamespace
 
         virtual bool GetIgnoreInputValidation() const;
         virtual void SetIgnoreInputValidation(const bool value);
+        #pragma deprecated(GetIgnoreInputValidation, SetIgnoreInputValidation)
```

Comment:
I think we can just remove this. It was never hooked up to anything so i would be surprised if anyone had a dependency on it. That said, UWP has a dependency on it :) so the build will fail there. You can either remove it from there too or I'd be fine with just checking it in broken to the feature branch and i can fix it when i do UWP.

Reply:
Removed, and updated tests